components/Results.js

```diff
@@ -60,17 +72,17 @@ export default class Results extends React.Component {
           //Case 3.1: There is definitely palm oil in there. Test with: 3700211234221
             return(
             <View style={styles.container}>
-              <Text style={styles.text}> {product_name} </Text>
-              <Text style={styles.text}> There are {palm_ingredients_total} ingredient from Palm oil in this </Text>
+              <Text style={styles.textName}> {product_name} </Text>
+              <Text style={styles.text}> This product contains {palm_ingredients_total} ingredient(s) from Palm oil</Text>
```

Comment:
I would like to see the components moved so that the animation sits between the text (it is a very small thing so no panic)